### ISSUE TITLE
Issue tab configuration

### ISSUE THREAD

**Issue by hutje** [Qrome/marquee-scroller#37]
When I press the configuration tab the screen inside the frame stays empty. Sometimes it appears and sometimes it shows only the half of the settings so I could not press the save button. I think it has something to do with handling multiple inputs at once. Like sending the html of the page and simultaneously keep the clock up to date.

**Comment by Qrome** [comment]
Yes, I have been struggling with it as well and been hitting some memory limitations in the expensive use of Strings in the code.   I believe I have the fix and it will be in version 2.2 -- you can check out that branch if you want to try it out.

**Comment by hutje** [comment]
Thanks. I have had the same issue. I have made a smart led controller which could be remotely controlled with a whole UI as browser page. But the string has its limitations when the RAM is full. So I have uploaded the html file to the spiffs of the esp8266 and let it send the html in parts to the client. Sending it in parts worked fine for me.

I saw another way to check the free ram and then calculate the safe length of the string to send without losing some data.
I recommend using the spiffs, but to upload to the spiffs, you need to prepare the Arduino IDE.
The spiffs can act like a sort of SD card.
